### DISCUSSION TITLE
Browserstack support for browser.mock

### DISCUSSION THREAD

**Discussion by Samic8** [webdriverio/webdriverio#6394] [Q&A] [answered]
Hi. Is there any timeline for when [browser.mock](https://webdriver.io/docs/mocksandspies.html) will be supported in browserstack?

Thanks!

**Comment by christian-bromann** [comment, marked as answer]
@Samic8 it is very unlikely that this will be ever implemented in BrowserStack. Mocking network capabilities rely on a different automation protocol (Chrome DevTools) which is not accessible in BS. That said, it might be very well implemented in Sauce Labs given that I can implement necessary components myself.

**Comment by Samic8** [comment]
Thanks @christian-bromann 

Are there any other packages that you know of that might work with browserstack?

I want to be able to intercept requests like browser.mock does without having to direct the client to a different endpoint (like with wiremock).
